Configuration.Override.props.in

```diff
@@ -6,11 +6,11 @@
       kept consistent with each other, lest Bad Things Happen™
       -->
     <!-- The default Android API level to bind in src/Mono.Android -->
-    <AndroidApiLevel>33</AndroidApiLevel>
+    <AndroidApiLevel>34</AndroidApiLevel>
     <!-- The Xamarin.Android $(TargetFrameworkVersion) value that corresponds to $(AndroidApiLevel) -->
-    <AndroidFrameworkVersion>v13.0</AndroidFrameworkVersion>
+    <AndroidFrameworkVersion>v14.0</AndroidFrameworkVersion>
```

Comment:
I think this is one of the things we can rip out now that we're no longer maintaining Classic in main.  (Not here and now, but eventually?)